OpenRA.Platforms.Default/Texture.cs

```diff
@@ -145,7 +179,8 @@ public void SetEmpty(int width, int height)
 
 			Size = new Size(width, height);
 			PrepareTexture();
-			OpenGL.glTexImage2D(OpenGL.GL_TEXTURE_2D, 0, OpenGL.GL_RGBA8, width, height,
+			var glInternalFormat = OpenGL.Features.HasFlag(OpenGL.GLFeatures.GLES) ? OpenGL.GL_BGRA : OpenGL.GL_RGBA8;
+			OpenGL.glTexImage2D(OpenGL.GL_TEXTURE_2D, 0, glInternalFormat, width, height,
```

Comment:
The methods were already copy-paste - but rather than adding to it I think extracting a helper for the boilerplate around calling `OpenGL.glTexImage2D` would be sensible.

Reply:
Fixed.